src/components/tree/phyloTree/renderers.js

```diff
@@ -374,7 +374,11 @@ export const branchStrokeForHover = function branchStrokeForHover(d) {
  * in practice, elements (or portions of elements) render outside this.
  */
 export const setClipMask = function setClipMask() {
-  const [xMin, xMax, yMin, yMax] = [...this.xScale.range(), ...this.yScale.range()];
+  const [yMin, yMax] = this.yScale.range();
+  // for the RHS tree (if there is one) ensure that xMin < xMax, else width<0 which some
+  // browsers don't like. See <https://github.com/nextstrain/auspice/issues/1755>
+  let [xMin, xMax] = this.xScale.range();
+  if (xMin>xMax) [xMin, xMax] = [xMax, xMin];
```

Comment:
Random thought: are you 100% sure `xMin` and `xMax` are *always* going to be number or do you need to do something like this to be safe:

```suggestion
  if (parseInt(xMin,10)>parseInt(xMax,10)) [xMin, xMax] = [xMax, xMin];
```

Reply:
It _should_ be - it's a bug if not - but I'll add this change to cover all bases. My hope is that TS would provide guarantees here, and so if there is a bug somewhere it'll identify it for us, but I've not had ~any experience with TS + d3 so time will tell!

Reply:
[Updated in force-push](https://github.com/nextstrain/auspice/compare/fcc3b68f23a2623de5ce0cbebf98ac2b5bda9c10..a17fb0b51a72ab0611b449bcc5e931fc23ea64b0)